docs/book/src/reference/markers/webhook.md

```diff
@@ -1,6 +1,6 @@
 # Webhook
 
-These markers describe how [webhook configuration](/TODO.md) is generated.
+These markers describe how [webhook configuration](../webhook-overview.md) is generated.
```

Comment:
The document should be: https://book.kubebuilder.io/reference/markers/webhook.html

Reply:
This doc is the marker reference (the URL you shared), so it seemed like this was supposed to go back to the overall webhook overview, correct?

Reply:
I wrote the webhook doc initially. The intent was to point to the webhook section of the marker doc, which was not available when I wrote it. We forgot to fix `TODO.md` when we got our marker document.

Reply:
I think I am definitely misunderstanding something. This file is:
docs/book/src/reference/markers/webhook.md
which is what ends up in:
https://book.kubebuilder.io/reference/markers/webhook.html

So if I'm understanding your suggestion correctly it would be a link to itself?